gluecodium/src/main/java/com/here/gluecodium/validator/LimePropertiesValidator.kt

```diff
@@ -63,6 +75,36 @@ internal class LimePropertiesValidator(private val logger: LimeLogger) {
             logger.error(limeProperty, "property in an interface cannot be marked with @Cpp(Ref)")
             result = false
         }
+        if (needsDocumentationComment(limeProperty, referenceMap) && !validateComments(limeProperty)) {
+            if (werrorPropertiesDocs) {
+                result = false
+            }
+        }
+        return result
+    }
+
+    private fun validateComments(limeProperty: LimeProperty): Boolean {
+        val propertiesDocsErrorMessage = "property must be documented; check 'docs/lime_markdown.md'"
+        var result = true
+
+        if (limeProperty.comment.isEmpty()) {
+            logger.maybeError(limeProperty, "main comment is empty; $propertiesDocsErrorMessage")
```

Comment:
I would make text more specific and detailed. "Main comment" is very unspecific. Also I would make full http link to the doc's section where it's described detailed. It make sense to add section in documentation with this particular warning which describes what went wrong and how to fix it.

Reply:
The documentation has been adjusted to contain the information about accepted syntax and conditions when warning/error occurs. I adjusted it for properties, functions and lambdas.

Moreover, `LimePropertiesValidator` has been adjusted to print more verbose warnings and contain the full reference to the docs. The user may use the link and open Gluecodium's GitHub.